### PR TITLE
[solidjs-pod] 2. Add social image and description

### LINKED ISSUE #270: [solidjs-pod] 2. Add social image and description
# Add framework social image and description

## Background

As an user I would like to have the possibility to share the site on social media and for it to have a nice image and description defined, so that it will increase its visibility and shareability

## Acceptance

- [x] Add metadata description
- [x] Add social image
- [ ] Test the site on "https://socialsharepreview.com/"

### PR DESCRIPTION
**Pull request by kodejuice**
## Type of change

Closes #270 

- [x] Content addition
- [ ] Bug fix
- [ ] Behavior change

## Summary of change

 Add a general social meta tags for frameworks.

### CODE REVIEW

**Comment by mbicknese** [comment]
I'm confused by the changes in this PR. I was expecting to see an image for SolidJS and maybe some meta info. But it adds images for other frameworks and a refactor instead.

Can you elaborate on what has changed?

File: packages/site/src/components/social-meta-tags.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { formatFieldName } from "@framework/system/src/util/string-utils"
+
+type SocialMetaTagsProps = {
+	title: string
+	siteName: string
+}
+
+export function SocialMetaTags({ siteName, title }: SocialMetaTagsProps) {
+	const description = `Search, compare, and discover top ${formatFieldName(
```
Comment (vyktoremario) [inline packages/site/src/components/social-meta-tags.tsx:9]: I thought this description is supposed to be in `packages/site/src/data/descriptions.ts` ?

**Review by vyktoremario — COMMENTED** [review]
Left a comment

**Comment by iansamz** [comment]
Hey @mbicknese & @vyktoremario . The changes in this PR were merged into main with this PR: [319](https://github.com/thisdot/framework.dev/pull/319). 

You can rebase this PR with main and add the description at `packages/site/src/data/descriptions.ts`

**Review by mbicknese — APPROVED** [review]

**Pull request merged**